docs/features.md

```diff
@@ -0,0 +1,54 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.Markdownlint addin.
+---
+The [Cake.Issues.Markdownlint addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from [Markdownlint] logfiles.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [MarkdownlintAddRuleUrlResolver] alias.
+
+# Supported log file formats
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.